test_util/summary.cpp

```diff
@@ -77,15 +78,19 @@ int main(int argc, char **argv) {
     int c                          = 0;
     bool reportStepsOnly           = false;
     bool listKeys                  = false;
+    bool noHeaders                 = false;
```

Comment:
this is mainly a preference thing but imho it is much more intuitive with enable than disable bools. ie bool headers = true, -n makes it false, no negation in ifs below.

Reply:
I fully agree with you. Copy past without thinking is not allways a good thing. Fixed now.